src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/PutExpressionInterestService.kt

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.services
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.stereotype.Component
+import software.amazon.awssdk.services.sqs.model.SendMessageRequest
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.exception.MessageFailedException
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.ExpressionOfInterest
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.ExpressionOfInterestMessage
+import uk.gov.justice.hmpps.sqs.HmppsQueue
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
+import uk.gov.justice.hmpps.sqs.eventTypeMessageAttributes
+import java.util.UUID
+
+@Component
+class PutExpressionInterestService(
+  private val hmppsQueueService: HmppsQueueService,
+  private val objectMapper: ObjectMapper,
+) {
+  private val eoiQueue by lazy { hmppsQueueService.findByQueueId("jobsboardintegration") as HmppsQueue }
+  private val eoiQueueSqsClient by lazy { eoiQueue.sqsClient }
+  private val eoiQueueUrl by lazy { eoiQueue.queueUrl }
+
+  fun sendExpressionOfInterest(expressionOfInterest: ExpressionOfInterest) {
```

Comment:
Also noticed this - should be sending to a topic and not a queue see [this](https://github.com/ministryofjustice/hmpps-integration-events/blob/beb2c3d219d67262ad286a3f20ee3338becb9cf1/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationevents/services/IntegrationEventTopicService.kt#L33)